app/assets/javascripts/global.js

```diff
@@ -172,13 +172,38 @@ function createTagSelect(tagType, selector, formType, scope) {
 
 function createSelect2(selector, options) {
   if (!options.width) { options.width = '100%'; }
-
-  if (options.ajax) {
-    options.ajax.delay = 200;
-    options.ajax.dataType = 'json';
-    options.ajax.cache = true;
-    if (!options.ajax.data) { options.ajax.data = queryTransform; }
-  }
-
+  if (options.ajax) addAjaxOptions(options);
```

Comment:
what... is this doing

Reply:
this is making code climate shut up, mostly. or did you mean what are the options themselves doing?

Reply:
I'm asking... uh.... why is there an if that doesn't have brackets and a method call on the same line and... stuff

Reply:
Oh. Javascript allows single line bracketless ifs. Do we not use that anywhere else, if that's a style thing I can make it match everything else.

Reply:
https://github.com/Marri/glowfic/issues/1435 but leaving it for now.